configure fails at librsvg-2.0 > 2.0

Attempting to compile on osx from master, via homebrew --HEAD fails at the configure step:

checking for librsvg-2.0 > 2.0... no
configure: error: Package requirements (librsvg-2.0 > 2.0) were not met:

No package 'librsvg-2.0' found




Shouldn't configure.ac say ">=" instead of ">" ?

--- a/configure.ac
+++ b/configure.ac
@@ -46,7 +46,7 @@ BOOST_OPTIONAL
 PKG_CHECK_MODULES([glibmm], [glibmm-2.4 >= 2.8])
 PKG_CHECK_MODULES([gdkmm], [gdkmm-2.4 >= 2.8])
 PKG_CHECK_MODULES([gerbv], [libgerbv >= 2.1.0])
-PKG_CHECK_MODULES([rsvg], [librsvg-2.0 > 2.0])
+PKG_CHECK_MODULES([rsvg], [librsvg-2.0 >= 2.0])
 
 AC_SUBST(gerbv_LIBS)
 AC_SUBST(gerbv_CFLAGS)

